gui/webdisplay/inc/ROOT/TWebWindow.hxx

```diff
@@ -143,6 +144,9 @@ public:
    /// Returns current number of active clients connections
    unsigned NumConnections() const { return fConn.size(); }
 
+   // Return connection id for given connection
+   unsigned GetConnectionId(unsigned num = 0) const;
```

Comment:
`unsigned` isn't really useful, unless you need to do bit manipulation or require overflow behavior. In all other cases please just use `int`.

Reply:
Why you prefer signed int?
Negative values are useless in case if I want to enumerate from 0 to N. 
This is like size_t or size_type

Reply:
Really. :-)

https://stackoverflow.com/questions/30395205/why-are-unsigned-integers-error-prone
https://news.ycombinator.com/item?id=2364741
https://github.com/ericniebler/stl2/issues/182

And yes, an unsigned `size_type` is seen as a mistake that we suffer from until today.